pkg/informant/agent.go

```diff
@@ -702,3 +764,45 @@ func (a *Agent) Resume(timeout time.Duration) error {
 
 	return nil
 }
+
+// SpawnRequestUpscale requests that the Agent increase the resource allocation to this VM
+func (a *Agent) SpawnRequestUpscale(timeout time.Duration, handleError func(error)) {
+	// Quick unregistered check
+	select {
+	case <-a.unregistered.Recv():
+		klog.Warningf(
+			"RequestUpscale called for Agent %s/%s that is already unregistered (probably *not* a race?)",
+			a.serverAddr, a.id,
+		)
+		handleError(context.Canceled)
```

Comment:
this seems like a weird sentinel error to pass around, but sure.

Reply:
just declare a sentinel error in this named of `ErrRequestConflictTimeout` or something similar.